media-gfx/freecad/freecad-0.19.2.ebuild

```diff
@@ -222,11 +222,22 @@ src_configure() {
 		# sub-packages will still be installed inside /usr/lib64/freecad
 		-DINSTALL_TO_SITEPACKAGES=ON
 
-		-DOCC_INCLUDE_DIR="${CASROOT}"/include/opencascade
-		-DOCC_LIBRARY_DIR="${CASROOT}"/$(get_libdir)
 		-DOCCT_CMAKE_FALLBACK=ON				# don't use occt-config which isn't included in opencascade for Gentoo
 	)
 
+	if has_version ">=sci-libs/opencascade-7.5"; then
```

Comment:
Pop in a bug reference?

Reply:
(I'll do it)

Reply:
Thanks @thesamesam, I went straight to sleeping after pushing the commit and didn't see the commit.